stylesheets/Senior-Basic.css

```diff
@@ -0,0 +1,327 @@
+    font-size: inherit !important;
+  }
+  h1, h2, h3, h4, h5 {
+    font-family: Helvetica Neue, Helvetica, Arial, sans-serif !important;
+    font-weight: 700 !important;
+  }
+  h1 {
+    font-size: 2.5rem !important;
+  }
+  @media(max-width:799px) {
+    h1 {
+      font-size: 1.75rem !important;
+    }
+  }
+  h2 {
+    font-size: 1.75rem !important;
+  }
+  @media(max-width:799px) {
+    h2 {
+      font-size: 1.4219rem !important;
+    }
+  }
+  h3 {
+    font-size: 1.3125rem !important;
+  }
+  @media(max-width:799px) {
+    h3 {
+      font-size: 1.1487rem !important;
+    }
+  }
+  h4 {
+    font-size: 0.9844rem;
+  }
+  @media(max-width:799px) {
+    h4 {
+      font-size: 2rem !important;
+    }
+  }
+  h5 {
+    font-size: 1rem;
+  }
+  @media(max-width:799px) {
+    h5 {
+      font-size: 0.875rem !important;
+    }
+  }
+  /* END Zeroheight style section */
+  /* Error style section */
+  .error-alert-item {
+    background-color: #F2DEDE;
+    border-left-width: .25rem;
+    border-left-color: #B3000F;
+    border-left-style: solid;
+    overflow: hidden;
+    position: relative
+  }
+  @media(max-width:799px) {
+    .error-alert-item {
+      border-left: 0;
+      border-right: 0
+    }
+  }
+  .error-alert-item__top {
+    display: flex;
+    padding-left: 3rem
+  }
+  .error-alert-item__top-text-container {
+    flex: 1 1 auto;
+    padding: 1rem
+  }
+  .error-alert-item__icon {
+    align-content: center;
+    left: 1.25rem;
+    pointer-events: none;
+    position: absolute;
+    top: 1.2rem;
+  }
+  .error-alert-item__effect {
+    font-size: 1.125rem;
+    font-weight: 700;
+    margin-bottom: .375em;
+    font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
+    line-height: 1.275em;
+    color: inherit
+  }
+  .error-alert-item__icon {
+    height: 2rem;
+    width: 2rem
+  }
+  /* Required text and regex error color */
+  .required-text, .requiredLabelEmail, .regexErrorLabel {
+    color: #B3000F !important;
+    font-style: normal;
+  }
+  /* END Error style section */
+  /* Buttons */
+  /* Button size and font */
+  .form-submit-button, .form-section-next, .form-section-prev, #divWorkflowContent .buttonPopup.blue {
+    font-family: Inter UI, Helvetica Neue, Helvetica, Arial, sans-serif !important;
+    font-weight: 500 !important;
+    text-align: center !important;
+    white-space: nowrap !important;
+    vertical-align: middle !important;
+    cursor: pointer !important;
+    -webkit-user-select: none !important;
+    -moz-user-select: none !important;
+    -ms-user-select: none !important;
+    user-select: none !important;
+    padding: .6rem 1.6rem !important;
+    font-size: 16px !important;
+    line-height: 1.5 !important;
+    border: 1px;
+    -webkit-border-radius: .25rem !important;
+    border-radius: .25rem !important;
+  }
+  /* Submit-Next button */
+  .form-submit-button, .form-section-next, /* Upload button */ #divWorkflowContent .buttonPopup.blue {
```

Comment:
Small suggestion here: Avoid in-line comments to help with readability. I suggest moving `/* Upload button */` to its own line and the upload button element selector on a new line below